config/config.go

```diff
@@ -15,11 +17,17 @@ type Option func(*ValidationOptions)
 // NewValidationOptions creates a new ValidationOptions instance with default values.
 func NewValidationOptions(opts ...Option) *ValidationOptions {
 	// Create the set of default values
-	o := &ValidationOptions{}
+	o := &ValidationOptions{
+		FormatAssertions:  false,
+		ContentAssertions: false,
+	}
 
 	// Apply any supplied overrides
 	for _, opt := range opts {
-		opt(o)
+		// Sanity
+		if opt != nil {
+			opt(o)
+		}
```

Comment:
Maybe this


```suggestion
		if opt == nil {
 			// Sanity
			continue
		}
		opt(o)
```

The sanity is in the nil check. The comment is misleading otherwise

Reply:
opted (no pun intended) to remove the comment but leave the logic as-is ... seemed more logical to me.